src/scss/partials/_nav.scss

```diff
@@ -197,6 +197,10 @@ $nav-popover-padding               : spacing(x2);
                     .c-header--transparent & {
                         color: $nav-text-color--transparent;
                     }
+
+                    .c-header--transparent .c-nav-popoverList & {
```

Comment:
Rather than do and tie it to the `c-nav-popoverlist`, you could just do:

```
.c-header--transparent & {
   color: $nav-text-color--transparent;

   &:hover {
  	 color: $grey--dark;
   }
}
```

Reply:
That rule applies to other header links as well, 
They should stay white.

Reply:
Ah ok – could you make it `color: inherit`, as I think the colour comes from the class above it anyway?